### PR TITLE
Adds wolfgirls

### PR DESCRIPTION
**Pull request by rye-rice**
## About The Pull Request

A long, long time ago, a kingdom was crumbling. It's royal family hath left them, and its citizens living in poverty.
Then a light came, a mysterious man took the throne, and attempted his best efforts to keep the kingdom alive.
Unfortunately, the kingdom was accursed to crumble, and no deal of effort could save it.
Despite this, though his efforts, the kingdom hath survived several months more had it not been for him.
The man did not ask for any rewards, except for one:
"add wolfgirls lmao"
And thus, he set forth a prophesy. Though the kingdom hath long faded, the prophesy still stands tall.
This brings us to today.

## Why It's Good For The Game

Jokes aside, it adds a pretty neat customization option, and just adds another option to felinids, speaking of which, @TheNeoGamer42 abhumans when

Screenshot_2715
Screenshot_2716

ported from https://github.com/Bungalow-13/Bungalow-13/pull/365
which was ported from https://github.com/MrMelbert/MapleStationCode/pull/112

@lunaboards-dev

## Changelog
:cl: Gofawful5
add: fox tails for felinids
refactor: demodularized sprite accesories, kind of
/:cl:

### CODE REVIEW

**Comment by TheNeoGamer42** [comment]
Dear god. Also abhumans aren't coming the funny people said that they just wanna shove the felinid stuff into humans lmao.

**Comment by TheNeoGamer42** [comment]
wait a fucking second this is a foxgirl tail not a wolfgirl tail i've been scammed, we've all been scammed

**Review by Kapu1178 — CHANGES_REQUESTED** [review]
Not a fan of the concept as a whole. Will bring it up with other maints.

I also shouldn't have to explain why the organ code added is bad. `cat/wolf`? Seriously?

File: code/modules/mob/dead/new_player/sprite_accessories.dm [hunk 1]
```diff
@@ -1721,6 +1721,16 @@
...
 
+/datum/sprite_accessory/tails/human/wolf
+	name = "Wolf"
+	icon_state = "wolf"
+	color_src = HAIR
+
+/datum/sprite_accessory/tails_animated/human/wolf
+	name = "Wolf"
+	icon_state = "wolf"
+	color_src = HAIR
```
Comment (francinum) [inline code/modules/mob/dead/new_player/sprite_accessories.dm:1732]: Duplicate set.

Comment (francinum) [inline code/modules/mob/dead/new_player/sprite_accessories.dm:1732]: Duplicate set.

File: code/modules/mob/dead/new_player/sprite_accessories.dm [hunk 2]
```diff
@@ -1721,6 +1721,16 @@
 	icon_state = "cat"
 	color_src = HAIR
 
+/datum/sprite_accessory/tails/human/wolf
+	name = "Wolf"
+	icon_state = "wolf"
+	color_src = HAIR
```
Comment (francinum) [inline code/modules/mob/dead/new_player/sprite_accessories.dm:1727]: Duplicate set.

File: code/modules/surgery/organs/tails.dm [hunk 3]
```diff
@@ -40,6 +40,27 @@
...
 
+/obj/item/organ/tail/cat/wolf
+	name = "wolf tail"
+	desc = "A severed wolf tail. Fiyaa!"
+	tail_type = "wolf"
+
+/obj/item/organ/tail/cat/wolf/Insert(mob/living/carbon/human/H, special = 0, drop_if_replaced = TRUE)
+	..()
+	if(istype(H))
+		var/default_part = H.dna.species.mutant_bodyparts["tail_human"]
```
Comment (francinum) [inline code/modules/surgery/organs/tails.dm]: Why is this here. It's the only thing differentiating it from the parent version.

File: code/modules/surgery/organs/tails.dm [hunk 4]
```diff
@@ -40,6 +40,27 @@
...
+			H.dna.features["tail_human"] = H.dna.species.mutant_bodyparts["tail_human"] = tail_type
+			H.update_body()
+
+/obj/item/organ/tail/cat/wolf/Remove(mob/living/carbon/human/H,  special = 0)
+	..()
+	if(istype(H))
+		H.dna.features["tail_human"] = "None"
+		H.dna.species.mutant_bodyparts -= "tail_human"
+		color = H.hair_color
+		H.update_body()
```
Comment (francinum) [inline code/modules/surgery/organs/tails.dm]: This entire proc is identical to the parent.

**Review by francinum — CHANGES_REQUESTED** [review]
This code is an absolute mess and desperately needs deduplication.

File: code/modules/mob/dead/new_player/sprite_accessories.dm (hunk 2 above)
Comment (francinum) [reply code/modules/mob/dead/new_player/sprite_accessories.dm:1727]: Eh, this enhances readability, it's fine.

File: code/modules/mob/dead/new_player/sprite_accessories.dm (hunk 1 above)
Comment (francinum) [reply code/modules/mob/dead/new_player/sprite_accessories.dm:1732]: Eh, this enhances readability, it's fine.

**Comment by 1glitchycent** [comment]
INCREDIBLY BASED.

**Comment by 1glitchycent** [comment]
> Not a fan of the concept as a whole. Will bring it up with other maints.
> 
> I also shouldn't have to explain why the organ code added is bad. `cat/wolf`? Seriously?

Cope.

File: code/modules/mob/living/carbon/human/species_types/felinid.dm [hunk 5]
```diff
@@ -61,6 +61,10 @@
 		if(H.dna.features["tail_human"] == "Cat")
 			var/obj/item/organ/tail/cat/tail = new
 			tail.Insert(H, drop_if_replaced = FALSE)
+		else if(H.dna.features["tail_human"] == "Wolf")
+			var/obj/item/organ/tail/cat/wolf/tail = new
+			tail.Insert(H, special = TRUE, drop_if_replaced = FALSE)
+			mutant_organs += /obj/item/organ/tail/cat/wolf
```
Comment (francinum) [inline code/modules/mob/living/carbon/human/species_types/felinid.dm:67]: Explain why this is necessary?

Comment (francinum) [reply code/modules/mob/living/carbon/human/species_types/felinid.dm:67]: ...Oh, I see what's up here.

Please restructure this to not result in an incorrect list entry.

**Review by francinum — CHANGES_REQUESTED** [review]
Prepare a savefile burn to rename felinids (and update their ID) to something more generic. 'Modified Human' per instance.

File: code/modules/mob/living/carbon/human/species_types/felinid.dm (hunk 5 above)
Comment (MarkSuckerberg) [inline code/modules/mob/living/carbon/human/species_types/felinid.dm:67]:
```suggestion
			mutant_organs = list(/obj/item/organ/tail/cat/wolf)
```
I think this is what is needed, but I could be wrong.

**Comment by MarkSuckerberg** [comment]
My verdict on this honestly is just to leave felinids as they are, and then hopefully someone goes the extra mile and actually folds them completely into humans. I just don't see the sense of doing two savefile burns, so, yeah.

**Comment by WindowsErrors** [comment]
_... the felinids will use any means ... commit in the new series of attacks against the so-called neighborhood sniper ..._

**PUP PUP PUP PUP PUP PUP**

Perched up high on a rooftop
Like a bird, I'm havin' evil thoughts
A black helm covers my face
And death flows through my mind at it's own pace
Sometimes I feel like a superhero
Urgin' to free my kind
I'm so accurate
I shoot ya heart with a dime
Now I'm back to the moral of my tale
I see a victim, and then my thoughts fail
My mind's still fumblin' with the present slavery, I'm here to save you but who's to save me
I hold my strap so long that my palms all wet
I put finger on the button and then the sight I check
I make the kill, here's the deal
After the trigger's pulled then the death is sealed
So it's the shots that sing out like a piper
Everyone's scared of the neighborhood sniper

So my boy is a furry
I put my gun up to his head
And simply scared the bitch, BOO
Being the E ya know I'm thinkin' devious thoughts
Take off ya clothes, nigga, and here comes a plot
His catbitch was scared, she made a move I pulled the trigger now the room is red
Now there's nothing left but a furry
Here's a four leaf clover punk make a wish
One **PUP** two **PUP** three strikes your out
This furry's dead now I'm lookin' for a window to get out
I made my escape the cops chase
Duckin' to an alley so they don't recognize my face
The boy had to pay the piper
So they all stay in fear of the neighborhood sniper

I'm doin' dirt by myself as I continue my mission
Killin' all furries as an addiction
As the E, lovin' the game, leavin' a shame
And in my heart I feel no muthafuckin' pain
Because my heart is cold as I was told
And with the A.K.47 I stand bold
Rippin' up shit with the fury of my gun shot
Pop pop pow as I watch the fur drop
Lovin' the site as the blood spill
Take a photograph, to keep in my hall in thrills 
I’m stayin' calm and never hyper
As I succeed in my mission
At bein' a neighborhood sniper

**Comment by TheNeoGamer42** [comment]
and so windowserror makes their rare appearance, shitposting upon the coder texts once more

**Comment by rye-rice** [comment]
Screenshot_284

**Pull request closed without merging**